Mathlib/Analysis/Complex/UpperHalfPlane/Manifold.lean

```diff
@@ -70,4 +71,28 @@ lemma mdifferentiable_iff {f : ℍ → ℂ} :
     fun h ⟨z, hz⟩ ↦ mdifferentiableAt_iff.mpr <| (h z hz).differentiableAt
       <| (Complex.continuous_im.isOpen_preimage _ isOpen_Ioi).mem_nhds hz⟩
 
+lemma mdifferentiable_num (g : GL(2, ℝ)⁺) :
```

Comment:
Could you also add the `ContMDiff` statements? It would feel weird to have only the differentiablity for functions which are in fact analytic.

Reply:
Sure, we can add that if you think it's needed. I'm curious why you view it as necessary though, because for complex manifolds all these versions of differentiability are equivalent to each other (a differentiable function is automatically analytic).

Reply:
I guess it can't do any harm, so I've added the `ContMDiff` counterparts of all these statements.

Reply:
I agree it won't be super useful, but still. When you will need to use a lemma (formulated over a general field) which requires higher smoothness, you will be happy that you do not have to do the dance going from differentiability to higher smoothness. And it's good for API completeness.